### PR TITLE
python3Packages.cloudscraper: init at 1.2.52

### PR DESCRIPTION
**Pull request by kini**
###### Motivation for this change

New package. Tests disabled for now since they pull in a lot of stuff. This PR depends on #107791.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/cloudscraper/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  propagatedBuildInputs = [
+    requests
+    requests-toolbelt
+    pyparsing
+  ];
+
+  # The tests require several other dependencies, some of which aren't in
+  # nixpkgs yet, and also aren't included in the PyPI bundle.  TODO.
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cloudscraper/default.nix:28]:
```suggestion
  doCheck = false;
  
  pythonImportsCheck = [ "cloudscraper" ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please change the title and commit message to ``python3Packages.cloudscraper: init at 1.2.50``

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107796` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>python37Packages.cloudscraper</li>
    <li>python38Packages.cloudscraper</li>
    <li>python39Packages.cloudscraper</li>
  </ul>
</details>

```
ERROR: Could not find a version that satisfies the requirement pyparsing>=2.4.7 (from cloudscraper)
ERROR: No matching distribution found for pyparsing>=2.4.7
```

File: pkgs/development/python-modules/cloudscraper/default.nix (hunk 1 above)
Comment (kini) [reply pkgs/development/python-modules/cloudscraper/default.nix:28]: Done, thanks for the tip.

**Comment by kini** [comment]
> Please change the title and commit message to `python3Packages.cloudscraper: init at 1.2.50`

Done (updated to 1.2.52).

Package builds on Python 3.7, 3.8, and 3.9, and PyPy 3. On Python 3.6, the dependency `cffi` is broken on master.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107796` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.cloudscraper</li>
<li>python38Packages.cloudscraper</li>
<li>python39Packages.cloudscraper</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107796` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.cloudscraper</li>
<li>python38Packages.cloudscraper</li>
<li>python39Packages.cloudscraper</li>
</ul>
</details>

**Pull request merged**